Index.html

```diff
@@ -0,0 +1,136 @@
+            <input
+            type="email"
+            placeholder="Enter your email addres"
+            name="email"
+            id="email"
+            readonly
+            />
+            <input id="submit" type="submit" value="Get Started" />          
+        </form>
+    </div> 
+
+</section>
+    	
+    
+    <div class="icon">
+        
+     <div>
+        <div class="Premium">
+            <div >
+                <img src="icon1.JPG" alt="" id="header-img">
+            </div>
+            <div>
+                <ul>
+               <h4 class="title"><a href="">Premium Materials</a></h4>
+               <p > Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase</p>  
+               </ul>
+            </div>
+        </div>
+             
+             <div class="Fast">
+                <div >
+                    <img src="icon2.JPG" alt="" id="header-img">
+                </div>
+                <div>
+                    <ul>
+                        <h4 class="title"><a href="">Fast Shipping</a></h4>
+                        <p > We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied</p>
+                     </ul>
+                </div>
+             </div>
+       
+        <div class="Quality">
+            <div >
+                <img src="icon3.JPG" alt="" id="header-img">
+            </div>
+                 <div>
+                     <ul>
+                    <h4 class="title"><a href="">Quality Assurance</a></h4>
+                    <p> For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument.</p>
+                  </ul>
+                </div>
+        </div>
+	            
+         </div>
+</div>
+    </ul>
+    </div>
+    <div>
+         <iframe href="https://youtu.be/y8Yv4pnO7qc" id="video" height="400" img src="Troman.JPG"   frameborder="0" allowfullscreen=""></iframe>
+    </div>
+  <section >
+
+    <div class="pricing">
+
+        <div class="level-container" >
+            <div class="level">
+                <h4><a >TENOR TROMBONE 3</a></h4>
+            </div>
+              <h2>$600</h2>
+                <p> Lorem ipsum.</p> 
+                <p> Lorem ipsum dolor</p> 
+                <p> Lorem ipsum dolor..</p>
+                <p> Lorem ipsum</p>
+             <input class="btn"  type="submit" value="Select"  background-color="red">
+     </div>
+      <div class="level-container" >
+        <div class="level">
+            <h4><a >BASS TROMBONE</a></h4>
+        </div>
+            
+            <h2>$900</h2>
+            <p> Lorem ipsum.</p>
+            <p> Lorem ipsum.</p>
+            <p> Lorem ipsum dolor.</p>
+            <p> Lorem ipsum.</p> 
+          <input  class="btn" type="submit" value="Select" class="btn">
+    </div>
```

Comment:
> Kindly change this size not similar to other cards and minimize the gap between cards

Great job @DamenaM , on suggesting some changes in this review, however make sure you explain the `WHY` of your comments, so the student understand why you request changes
**Example of Good request**: 
```bash
Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible
```